test/parallel/test-v8-serdes.js

```diff
@@ -133,13 +138,11 @@ const objects = [
 }
 
 {
-  assert.throws(
-    () => { v8.Serializer(); },
-    /^TypeError: Class constructor Serializer cannot be invoked without 'new'$/
-  );
-
-  assert.throws(
-    () => { v8.Deserializer(); },
-    /^TypeError: Class constructor Deserializer cannot be invoked without 'new'$/
-  );
+  assert.throws(() => {
```

Comment:
I'm with @thefourtheye, also could drop the `() => { }` to get:
```js
assert.throws(v8.Serializer, serializerTypeError);
```
Now that's clean ✨